### PR TITLE
Jormungandr: revert unicode literals

### PR DESCRIPTION
**Pull request by pbougue**
This is a partial revert (almost complete) of https://github.com/CanalTP/navitia/pull/2337

This should be way easier to read commit by commit.

Although nothing new crashed or bugged, we choose to revert that change as the impact on our I/O is very difficult to assess.

### CODE REVIEW

**Review by kinnou02 — APPROVED** [review]

File: source/tyr/tests/docker_wrapper.py [hunk 1]
```diff
@@ -46,7 +45,7 @@ def _get_docker_file():
     is reduced by 10s
     """
     from io import BytesIO
-    return BytesIO('FROM {}'.format(POSTGRES_IMAGE).encode('utf-8'))
+    return BytesIO('FROM {}'.format(POSTGRES_IMAGE))
```
Comment (mbounabi) [inline source/tyr/tests/docker_wrapper.py]: str?

File: source/tyr/tests/integration/users_test.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def test_add_user_without_shape(mock_rabbit):
     resp = api_post('/v0/users/', data=data, content_type='application/json')
 
     def check(u):
-        gen = (k for k in user if k is not 'shape')
+        gen = (k for k in user if k != 'shape')
```
Comment (mbounabi) [inline source/tyr/tests/integration/users_test.py:183]: ?

File: source/tyr/tests/integration/users_test.py [hunk 3]
```diff
@@ -221,7 +221,7 @@ def test_add_user(mock_rabbit, geojson_polygon):
     resp = api_post('/v0/users/', data=data, content_type='application/json')
 
     def check(u):
-        gen = (k for k in user if k != 'shape')
+        gen = (k for k in user if k is not 'shape')
```
Comment (mbounabi) [inline source/tyr/tests/integration/users_test.py]: ?

File: source/tyr/tyr/api.py [hunk 4]
```diff
@@ -30,8 +30,9 @@
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
 
-from __future__ import absolute_import, print_function, unicode_literals, division
-from tyr import resources, app, api
+from tyr import resources
+
```
Comment (mbounabi) [inline source/tyr/tyr/api.py]: Why?

File: source/tyr/tests/integration/users_test.py (hunk 2 above)
Comment (TeXitoi) [inline source/tyr/tests/integration/users_test.py:183]: Why this change...

File: source/tyr/tests/integration/users_test.py (hunk 3 above)
Comment (TeXitoi) [inline source/tyr/tests/integration/users_test.py]: ... And then the exact contrary?
